pkg/querier/blocks_consistency_checker.go

```diff
@@ -37,21 +37,15 @@ func NewBlocksConsistencyChecker(uploadGracePeriod, deletionGracePeriod time.Dur
 		}),
 		checksFailed: promauto.With(reg).NewCounter(prometheus.CounterOpts{
 			Name: "cortex_querier_blocks_consistency_checks_failed_total",
-			Help: "Total number of consistency checks failed on queried blocks.",
+			Help: "Total number of consistency checks failed on queried blocks. A failed consistency check means that some of the block which had to be queried weren't present in any of the store-gateways.",
```

Comment:
[nit] Would this be clearer?

```suggestion
			Help: "Total number of queries that failed consistency checks. A failed consistency check means that some of at least one block which had to be queried wasn't present in any of the store-gateways.",
```

Reply:
I'm wondering if we should also clarify the description on the `cortex_querier_blocks_consistency_checks_total` metric to match the new meaning.